turn-lsp/org.jucmnav.turn.diagram/src/main/java/org/jucmnav/turn/diagram/TurnDiagramModule.xtend

```diff
@@ -21,8 +21,9 @@ class TurnDiagramModule extends IdeDiagramModule {
 		TurnLayoutEngine
 	}
 	
+	//TODO: swap TurnDiagramGenerator with TurnDiagramOtherGenerator
```

Comment:
TODO: Remove TODO; 
Also, rename "TurnDiagramOtherGenerator"; Suggestion: "TurnDiagramGenerator" We handle the arbitration of TGRL vs TUCM further in.